portal/patient/fwk/libs/verysimple/Authentication/PassPhrase.php

```diff
@@ -23,7 +23,7 @@ class PassPhrase
      */
     static function GetRandomPassPhrase($length = 8)
     {
-        srand((double) microtime() * 1000000);
+        mt_srand((double) microtime() * 1000000);
```

Comment:
i know this is just in draft from, but I can't stop myself from looking at any PRs having to do with security/random generators :)

I am guessing the above line can be removed since no need to seed